docs/Getting Started.md

```diff
@@ -0,0 +1,92 @@
+# Getting Started
+Welcome to Inductiva, a Python API package that simplifies running large-scale 
+physical system simulations in the cloud. Just few lines of code unleash its potential, freeing 
+you from local hardware limits and the technical hassles of scaling simulations. 
+
+### What We'll Cover:
+
+* [Inductiva API Installation]()
+* [What to read next]()
+* [Troubleshooting]()
+* [Frequently Asked Questions (FAQs)]()
+* [Useful Terminology]()
+
+## Inductiva API Installation
+
+To install the latest Inductiva package release on PyPI with pip, just open your 
+terminal and run:
+
+```
+pip install --upgrade inductiva
+```
+---
+
+> **Tip:** To ensure smooth installation and operation of our API, we recommend 
+keeping your Python package manager, Pip, up-to-date. Visit [Pip's official installation guide](https://pip.pypa.io/en/stable/installation/) for detailed instructions on updating Pip.
+
+
+Encountering issues? Don’t worry. Head over to [Troubleshooting]() to work it out.
+
+Otherwise, you're all set! You can now decide [what to read next]().
+
+## What to read next
+
+You can start experimenting with Inductiva by choosing from a variety of ready-to-use, 
+[open-source simulators]() that are immediately available for use on our default 
+machines.
+
+You can set up custom, dedicated machine groups for more complex or [larger-scale simulation]() 
+tasks, tapping into the more extensive [computational resources]() of Inductiva.
+
+You can get inspired by how other researchers have used Inductiva API to configure, 
```

Comment:
Do we have this? @IvanPombo ?

Reply:
Or is this about the other repos? (it could be!)

Reply:
so this section should be able to act as a standalone that would link you to different parts of the repo, could be other projects. I used the play of words but can make it more explicit with titles. 
so the logic or journey is:
1- you understand and learn what inductiva is and who it is for
2- you install it 
3- you then go to "what to read next" to choose what you want to do with it. 
it is for ease of navigation. looked like good practice on other repos.